### PR TITLE
Fix FFT

### PR DESCRIPTION
**Pull request by ericpre**

### CODE REVIEW

File: hyperspyui/plugins/fft.py [hunk 1]
```diff
@@ -57,287 +54,194 @@ def create_actions(self):
...
+        roi_signal = roi.interactive(s, recompute_out_event=False)
+        s_roi_fft = hs.interactive(roi_signal.fft,
+                                   event=roi.events.changed,
+                                   recompute_out_event=False,
+                                   shift=shift)
+
+        s_roi_fft.plot(power_spectrum=power_spectrum)
+
+    # def live_fft(self, signals=None):
+    #     # Old broken version
```
Comment (vidartf) [inline hyperspyui/plugins/fft.py]: This is the purpose of VCS ;)

File: hyperspyui/plugins/fft.py [hunk 2]
```diff
@@ -57,287 +54,194 @@ def create_actions(self):
                         "navigator",
                         selection_callback=self.ui.select_signal)
 
-        self.add_action('nfft', "Signal FFT", self.nfft,
-                        icon='nfft.svg',
-                        tip="Perform a fast fourier transform on the " +
-                            "entire signal",
-                        selection_callback=self.ui.select_signal)
```
Comment (vidartf) [inline hyperspyui/plugins/fft.py]: Not useful to have the full ND FFT after all? :)

Comment (ericpre) [reply hyperspyui/plugins/fft.py]: This is what the FFT in hyperspy does.

File: hyperspyui/plugins/fft.py (hunk 1 above)
Comment (ericpre) [reply hyperspyui/plugins/fft.py]: Yes, done!

File: hyperspyui/plugins/fft.py (hunk 2 above)
Comment (vidartf) [reply hyperspyui/plugins/fft.py]: Ah, neat.

**Comment by ericpre** [comment]
@francisco-dlp, would you some time to have a quick look at this to get it in the next release?
It is not as good as I would it to be, but this is an incremental progress. I would suggest to merge this and improve it further in a future PR.

**Comment by ericpre** [comment]
The test suite on python 3.5 on appveyor times out because with the current conda version (4.7.10), the environment cannot solved for python 3.5.

**Pull request merged**